composer.json

```diff
@@ -1,24 +1,26 @@
 {
     "name": "codeception/module-doctrine2",
     "description": "Doctrine2 module for Codeception",
-    "keywords": [ "codeception", "doctrine2" ],
-    "homepage": "https://codeception.com/",
-    "type": "library",
     "license": "MIT",
+    "type": "library",
+    "keywords": [
+        "codeception",
+        "doctrine2"
+    ],
     "authors": [
         {
-            "name":"Michael Bodnarchuk"
+            "name": "Michael Bodnarchuk"
         },
         {
-            "name":"Alex Kunin"
+            "name": "Alex Kunin"
         }
     ],
-    "minimum-stability": "dev",
+    "homepage": "https://codeception.com/",
     "require": {
         "php": "^8.0",
-        "ext-pdo": "*",
         "ext-json": "*",
-        "codeception/codeception": "^5.0.0-alpha1"
+        "ext-pdo": "*",
+        "codeception/codeception": "dev-5.0-interfaces"
```

Comment:
please use 'as 5.0.0' as we may need to fake this constraint when installing dependencies in another repository, for example `module-symfony`.